webpack/published-js-components.js

```diff
@@ -31,7 +31,7 @@ const publishedJSComponents = {
   GridKeyboardNavigationContext: "components/GridKeyboardNavigationContext/GridKeyboardNavigationContext.js",
   allIcons: "components/Icon/Icons",
   Skeleton: "components/Skeleton/Skeleton.jsx",
-  AvatarGroup: "components/AvatarGroup/AvatarGroup.jsx",
+  Slider: "components/Slider/Slider.jsx",
```

Comment:
I think this is exported both here and [the ts components](https://github.com/mondaycom/monday-ui-react-core/pull/931/files#diff-6e573898e28e78052777e024100430d238e4c9bfdc2fb561c360a075e2fa7de2R32)